src/libsync/theme.cpp

```diff
@@ -220,19 +226,29 @@ QString Theme::themeImagePath(const QString &name, int size, bool sysTray) const
 {
     const auto flavor = (!isBranded() && sysTray) ? systrayIconFlavor(_mono) : QLatin1String("colored");
 
-    // branded client may have several sizes of the same icon
-    const QString filePath = (isBranded() && size > 0)
-            ? QString::fromLatin1(":/client/theme/%1/%2-%3").arg(flavor).arg(name).arg(size)
-            : QString::fromLatin1(":/client/theme/%1/%2").arg(flavor).arg(name);
+#ifdef APPLICATION_ICON_SET
+    const auto useSvg = QByteArray(APPLICATION_ICON_SET).toUpper() == QByteArrayLiteral("SVG");
```

Comment:
@er-vin Since the code is almost the same and I've seen it 3 times per unit, would you consider moving it out to a function in the anonymous namespace so it takes **isBranded** as an argument?
Like:
```
static bool useSVG(bool isBranded)
{
#ifdef APPLICATION_ICON_SET
    returnQByteArray(APPLICATION_ICON_SET).toUpper() == QByteArrayLiteral("SVG");
#else
    return !isBranded;
#endif
}
```

Reply:
Yeah thought about extracting in a function but didn't do it because the branding would impact similarly every time... Not sure I like such a function where we pass as branded parameter to it... I might have a compromise, I'll try to implement it and see how that looks.

Reply:
OK... found no construct I like, but... it occurs to me that once this PR is in, there would be no good reason to have APPLICATION_ICON_SET as undefined, so it might be better to fail the build in that case instead of trying to have a kind of fallback.

Reply:
@er-vin Approved. Also, I've been thinking, if APPLICATION_ICON_SET is constant for the entire build, you could have just used a constant that gets assigned once, and get rid of a function. But that's up to you.

Reply:
Well, I'd rather not, I'm not a huge fan of having stuff which allocates in a static const (and unfortunately we can't go constexpr since QByteArray will allocate.